feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/components/TimetableItemDetailHeadline.kt

```diff
@@ -66,8 +68,10 @@ fun TimetableItemDetailHeadline(
         FlowRow {
             TimetableItemTag(
                 modifier = Modifier.align(Alignment.CenterVertically),
-                tagText = timetableItem.room.name.currentLangTitle,
-                tagColor = roomTheme.primaryColor,
+                tagText = timetableItem.room.name.currentLangTitle.toUpperCase(Locale.current),
```

Comment:
It seems I forgot to convert the text on the timetable details screen to uppercase. 🙏